src/request.rs

```diff
@@ -11,11 +11,21 @@ use crate::{
     AData, ADataAddress, ADataAppend, ADataIndex, ADataOwner, ADataPubPermissions,
     ADataUnpubPermissions, ADataUser, AppPermissions, Coins, IDataAddress, IDataKind, MDataAddress,
     MDataPermissionSet, MDataSeqEntryActions, MDataUnseqEntryActions, PublicKey, SeqMutableData,
-    UnseqMutableData, XorName,
+    Signature, UnseqMutableData, XorName,
 };
 use serde::{Deserialize, Serialize};
 use std::fmt;
 
+pub type TransactionId = u64; // TODO: Use the trait UUID
+
+#[derive(Hash, Eq, PartialEq, PartialOrd, Ord, Clone, Serialize, Deserialize)]
+pub struct NewAccount {
+    pub destination: XorName,
+    pub owner: PublicKey, // deterministically created from passwords
```

Comment:
I wonder if we could call this something like `authorised_getter` instead to make the intent clearer?  Because this is the transient key used just for logging in really, I don't see it as representing the owner as much as his other identities with coin balances attached, etc.  I'm not sure though - it's certainly not a blocker.